pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -993,20 +989,6 @@ def update_device_type_if_training_type_plugin_passed(self) -> None:
                 elif self.has_gpu:
                     self._device_type = DeviceType.GPU
 
-    @property
-    def is_slurm_managing_tasks(self) -> bool:
-        rank_zero_deprecation(
-            "`AcceleratorConnector.is_slurm_managing_tasks` was deprecated in v1.5 and will be removed in v1.6."
-        )
-        return self._is_slurm_managing_tasks
-
-    @is_slurm_managing_tasks.setter
-    def is_slurm_managing_tasks(self, value: bool) -> bool:
-        rank_zero_deprecation(
-            "`AcceleratorConnector.is_slurm_managing_tasks` was deprecated in v1.5 and will be removed in v1.6."
-        )
-        self._is_slurm_managing_tasks = value
-
```

Comment:
You only need to remove this code, the public methods. 
The protected attribute needs to stay and will be taken care of in my PR here #10102
It is part of a larger simplification and improvement.

Reply:
Done, wasn't aware that I just had to remove these

Reply:
Yep, I fully understand

Thanks a lot for taking this <3